### PR TITLE
User Story 14: market

### PR DESCRIPTION
**Pull request by Mistake91**
i did some changes and now there are 2 buttons, one for buying and one for selling

### CODE REVIEW

**Review by shebtastic — CHANGES_REQUESTED** [review]
i have noticed that all components in your project are have lowercased filenames.
please update these to conform to conventions.

something for a future refactoring could be, that you might want to take a look at shared code among your pages and components and refactor these into reusable functions. this might be especially interesting for recurring tasks like the resource gathering or buying stuff from your market.

File: components/buy.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import { useState } from "react";
+
+export default function Buy({ inventory, setInventory }) {
+  const [choosedItem, setChoosedItem] = useState("");
+  const [choosed, setChoosed] = useState(false);
+
+  function buyItem(amount) {
+    console.log(inventory.dwarfi.amount);
+    console.log(choosedItem);
+    console.log(amount * 2);
```
[the remark this answers is not in the document]
Comment (Mistake91) [reply components/buy.js]: thanks, forget them

File: components/inventory.js [hunk 2]
```diff
@@ -3,37 +3,54 @@ import Image from "next/image";
...
-        <InventoryPlace key={item.id} row={item.row} column={item.column}>
-          <Image src={icon_placeholder} alt="" width={40} height={40} />
-          <StyledP>{item.name}</StyledP>
-          <StyledP>{item.amount}</StyledP>
-        </InventoryPlace>
-      ))}
+      <StyledUL>
+        {Object.values(filteredInventory).map(
+          (item) =>
+            item.id != 99 && (
```
[the remark this answers is not in the document]
Comment (Mistake91) [reply components/inventory.js]: changed it, thanks

File: components/layout.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import styled from "styled-components";
 import { useRouter } from "next/router";
 import { useState } from "react";
 
-import { Grid } from "@/components/homeStyles";
```
[the remark this answers is not in the document]
Comment (Mistake91) [reply components/layout.js:5]: yea thats right, moved it :)

File: components/inventory.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mistake91) [reply components/inventory.js]: and yes, i will create a user story where i will do some refactoring or is it important to do it now? 👀

**Review by NikRoe — APPROVED** [review]
Hi there,

looks good to me

**Pull request merged**